modules/phase0/src/main/resources/phase0/Phase0_Instrument_Matrix - Spectroscopy.tsv

```diff
@@ -190,34 +190,6 @@ GMOS-S	B1200 1.0" RG610	singleslit,multislit	1.0"	1.000	330	B1200	RG610	0.61	1.0
 GMOS-S	B1200 1.5" RG610	singleslit,multislit	1.5"	1.500	330	B1200	RG610	0.61	1.03	0.463	0.159	1248	no		s
 GMOS-S	B1200 2.0" RG610	singleslit,multislit	2.0"	2.000	330	B1200	RG610	0.61	1.03	0.463	0.159	936	no		s
 GMOS-S	B1200 5.0" RG610	singleslit,multislit	5.0"	5.000	330	B1200	RG610	0.61	1.03	0.463	0.159	374	no		s
-GMOS-S	B600 0.25"	singleslit,multislit	0.25"	0.250	330	B600	none	0.36	0.72	0.461	0.307	3376	no		s

```

Comment:
This file presumably comes from science.  They should update their source similarly.